frontend/src/RecordingPageComp/RecordingComp.jsx

```diff
@@ -0,0 +1,61 @@
+import { AudioRecorder, useAudioRecorder } from "react-audio-voice-recorder";
+import { useContext } from "react";
+import { UserContext } from "../UserContext.js";
+
+function RecordingComp() {
+  const { user } = useContext(UserContext);
+
+  const recorderControls = useAudioRecorder();
+  const reader = new FileReader();
+
+  const addAudioElement = (blob) => {
+    reader.readAsDataURL(blob);
+    const src = URL.createObjectURL(blob);
+    setAudioSource(src);
+
+    reader.onload = function (event) {
+      const audiodata = event.target.result.split(",")[1]; //Stores part of URL after the ,
```

Comment:
It would be safest to add some error handling if there is no "," in result. 
Currently it would throw index out of bound.